pkg/visitors/assemble.go

```diff
@@ -133,6 +134,15 @@ func (v *Assemble) Visit(f uefi.Firmware) error {
 		// TODO: handle the whole header instead of doing this
 		binary.LittleEndian.PutUint64(fBuf[32:], f.Length)
 
+		// Write the correct GUID to the correct spot
```

Comment:
Can you add a note here saying to lookup the meaning of EFI_FIRMWARE_FILE_SYSTEM3_GUID in section 3.2.2, volume X in the PI spec version Y?

Reply:
Done